lib/graphql/define/defined_object_proxy.rb

```diff
@@ -10,6 +10,11 @@ def types
         GraphQL::Define::TypeDefiner.instance
       end
 
+      # TODO: do I actually want to add this?
```

Comment:
I asked because I'm always a bit wary with exposing key/value stores like that. lol

> it's more prone to typos, that's a bit sad

Right!

```ruby
MyType = GraphQL::ObjectType.define do 
  required_feature_flag :top_secret_access 
end
```

Letting people expose this on their own via `accepts_definitions` felt less prone to errors (and typos). It also means we don't need to expose `metadata` in the `define` block.

I could be wrong though. :)